### ISSUE TITLE
/metrics endpoint

### ISSUE THREAD

**Issue by brujoand** [Smashing/smashing#152]
**Is your feature request related to a problem? Please describe.**
I would like to persist data in prometheus. For that to work I would need a /metrics endpoint.

**Describe the solution you'd like**
Allow dashboards to be text only, for instance json or yaml.

**Describe alternatives you've considered**
Persisting to a database and scraping that with prometheus, manually scarping the dashboard html with regexes and extending smashing to push data to a prometheus sidecar.

**Notes**

Basically, I want to be able to create a dasboard which only outputs json, where thes json is the values used in my dashboards.

**Comment by atilleh** [comment]
Hello,

Sounds like the `/events` route: https://github.com/Smashing/smashing/blob/master/lib/dashing/app.rb#L76, this solution should be enough. 
(usable through : http://your_dashboard:3030/events

Does it seem correct to you ?
